app/models/event.rb

```diff
@@ -7,8 +7,8 @@ class Event < ApplicationRecord
   belongs_to :organisation
 
   scope :upcoming, lambda { |n|
-                               where('start_date > ?', DateTime.current.midnight)
-                              .order('created_at DESC')
+                               where('start_date >= ?', Time.zone.now.to_date)
```

Comment:
IMO this seems to be out of scope for this PR as it does not involve searching for the events, we can merge it in though :smiley:

Reply:
@sigu if you try removing this functionality you will realize why I have change it :+1: